discourse-doctor.rb

```diff
@@ -1,18 +1,26 @@
 #!/usr/bin/env ruby
+# coding: utf-8
 
 Dir.chdir("/") do
   require 'bundler/inline'
   require 'net/http'
   require 'net/smtp'
+  require 'pathname'
 
   gemfile(true, ui: false) do
     source 'https://rubygems.org'
 
     gem 'dnsruby'
     gem 'paint'
+    gem 'sys-filesystem' # for checking available disk space
```

Comment:
Where did you find the `-b` option? It doesn't exist in my locally installed `df` or in the `df` which comes with the default ubuntu container. An that goes into the direction I am warning against. I would use a library which makes my intention clear and which version and API I can pin down exactly instead of depending on parsing the textual output of a command which may change at some time I cannot control.

But it's not that important. Being in a docker container, it should be relatively save. I think you mean the `-B512` option? Are you okay with parsing the output in ruby instead of using tail and awk?

```ruby
bytes_available = `df -B512 #{DISCOURSE_PATH}`.lines[1].split(" ")[3].to_i * 512
```